Design improvements(?)

bilgeonceken opened the pull request:
![comparison](https://cloud.githubusercontent.com/assets/15635245/23000352/d7c46e62-f3e8-11e6-9896-aed8abbf64a8.jpg)

bilgeonceken committed:
Design improvements(?)

libduck2 commented:
On a first sight, not a deep inspection:

- The red is too oibs 
- I would resist dropping the depressive and succinct style(capital letters?!)

However the form default value thing and the current course wrapper looks nice.

libduck2 commented:
perhaps this needs a CSS reset, wtf sans serif font is this destroys the overall "my sun is fading" attitude

libduck2 commented:
I won't deny this PR exposes some design deficiencies, but introducing an Ubuntu/OIBS red, rounding the edges and capitalizing first letters don't look necessarily userfriendly to me.

Issues I notice are:
- in Firefox, text, form, buttons and boxes are not really aligned
- a CSS reset is desperately needed because of the goofy button styling some OS/browsers use
- a border would be nice for "current courses"
- course colors are too much of one saturation- violet/blue/green

also a grid and headers for the table could be good. a SVG grid maybe?

bilgeonceken committed:
New theme with (almost)brand new css
Used skeleton as boiler plate. Did not touch course blocks' color palette.
"About" and "Wrong" related parts are not included in index.html and commented out on main.js(marked with TODOs).

kambafca commented:
![newstyle](https://cloud.githubusercontent.com/assets/15635245/23048358/f3e84070-f4bd-11e6-8378-856b757acc3a.png)

libduck2 commented:
I don't think there is a point in selecting a theme now, let's first do the layout and then the colors.
More issues are:

- basbursen abomination keeps next/prev buttons aligned with the page on mobile, we don't
- flex wrapping can be unreliable, perhaps migrate to [PureCSS](purecss.io)? I think that will result in an actually smaller style.css. it is a mess now.

duck2 commented on index.html:
```diff
@@ -1,84 +1,111 @@
-<a id="about-link">about</a>
-<a id="wrong-link">wrong!</a>
-</div>
+<html lang="en">
```
there is no point serving those, the browser injects them. see google's [style guide](https://google.github.io/styleguide/htmlcssguide.xml).

duck2 commented on index.html:
```diff
@@ -1,84 +1,111 @@
-<script src="awesomplete.js"></script>
-<script src="data.js"></script>
-<script src="main.js"></script>
+  <!-- FONT
```
let's keep the HTML comment-free please. it should be self explanatory.

duck2 commented on index.html:
```diff
@@ -1,84 +1,111 @@
-<script src="main.js"></script>
+  <!-- FONT
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
```
let's keep the HTML webfont-free please. system fonts are enough for most things, they download 0kb and they don't jitter or show blank text while the actual font is being downloaded.

duck2 commented on index.html:
```diff
@@ -1,84 +1,111 @@
+        <div class="stuff">
+            <div class="options">
+                <div>Department: <input style="position:/*
+.eatd{position: relative; height: 100%; cursor: pointer;}*/relative; left:44px;" id="dept" class="" placeholder="EE, MINE etc."></div>
```
no px-oriented and/or long inline styling please. most CSS frameworks have stuff for aligning forms.

duck2 commented on style.css:
```diff
@@ -1,93 +1,98 @@
-	height: 100%;
-	font-family: sans-serif;
-	font-size: 10pt;
+header{
```
about the header- it is designed be small and fit on mobile screens without requiring flex wrap. this layout won't fit on mobile screens and will look weird. try resizing the browser below 400px width.

duck2 commented on style.css:
```diff
@@ -102,77 +107,171 @@ td{min-width: 8em; border: 1px solid black; height: 25em;}
+    bottom: 5px;
+}
+
+/* COPY PASTE FROM LIBDUCK. PROPABLY RELATED WITH SOME THINGS*/
```
I know this is not an actual comment but let's keep the CSS comment free too. anlayan anlasın anlamayan karpuz yesin aq

kambafca commented:
I should not have pushed to this branch. Not only i hated it but i also destroyed the consistent flexbox structure after i added(copy pasted) the styling for course list to make a workaround for something that does not need a workaround kind of solution.  Working at night time is bad. I'll try to create something else if i have time. border-collapse is buggy by the way. Borders disappear sometimes.

kambafca commented:
PureCSS +1

bilgeonceken committed:
New color palette and code clean up

kambafca commented:
![out](https://cloud.githubusercontent.com/assets/15635245/23093151/b668d5b2-f5e3-11e6-89e7-667ebaffec2a.jpg)

bilgeonceken committed:
ops. minoe commit

bilgeonceken committed:
No more inline styling

bilgeonceken committed:
header buttons wraps no more

kambafca commented:
Nevermind the commit. I still hate it

libduck2 commented:
colors intensify hate but hard to change when they are coupled with layout xd

I'll push a clean slate purecss layout if I can find enough time and motivation.

aybberk commented:
bence bilgenin dizaynları so gay :)
